config/locales/en.yml

```diff
@@ -1109,6 +1109,15 @@ en:
         sent_at: Sent at %{datetime}
       title: Invitations
   messages:
+    closing_soon:
+      email:
+        body: 'We are closing GetYourRefund for the tax season on October 15th. We will need all required documentation by October 8th; otherwise, we will not be able to prepare your taxes. Please login and respond by October 8th. If we do not hear from you by then, we will close out your case and you can file with us next year.
```

Comment:
Dumb question - are we sure that we want the trailing newlines in this string? Not a blocker; just curious.

```
irb(main):003:0> I18n.t("messages.closing_soon.email.body", locale: "en").last(1)
=> "\n"
```

Reply:
This is what normalize does. I'm not sure why I can't use the pipe to create a multiline string without trailing whitespace now that we're using normalize.